### PR TITLE
Added an extra check for numbers to make sure that we could also use oauth token expiration as var

### PR DESCRIPTION
**Pull request by andralungu**
@atrifan ping.

### CODE REVIEW

**Review by atrifan — APPROVED** [review]

**Review by bogdanciuca — APPROVED** [review]
👍

File: src/lua/api-gateway/validation/validatorsHandler.lua [hunk 1]
```diff
@@ -241,7 +241,7 @@ end
 function ValidatorsHandler:saveContextInRequestVars(localContext)
      for k,v in pairs(localContext) do
     -- for i,k in pairs(varsToSet) do
-        if ngx.var[k] ~= nil and type(localContext[k]) == "string" then
+        if ngx.var[k] ~= nil and (type(localContext[k]) == "string" or type(localContext[k]) == "number") then
```
Comment (corinapurcarea) [inline src/lua/api-gateway/validation/validatorsHandler.lua:244]: You could use v instead of localContext[k]

**Review by corinapurcarea — CHANGES_REQUESTED** [review]

File: src/lua/api-gateway/validation/validatorsHandler.lua (hunk 1 above)
Comment (corinapurcarea) [reply src/lua/api-gateway/validation/validatorsHandler.lua:244]: @andralungu , @atrifan, @bogdanciuca  WDYT if we also check for boolean?

Comment (bogdanciuca) [reply src/lua/api-gateway/validation/validatorsHandler.lua:244]: @corinapurcarea Are you aware of any `boolean`s currently being set on `ctx`, or just planning ahead?

Comment (corinapurcarea) [reply src/lua/api-gateway/validation/validatorsHandler.lua:244]: planning ahead

Comment (atrifan) [reply src/lua/api-gateway/validation/validatorsHandler.lua:244]: @corinapurcarea it's ok

**Pull request merged**